src/libraries/System.Runtime.Numerics/src/System/Numerics/BigIntegerCalculator.FastReducer.cs

```diff
@@ -27,19 +27,19 @@ public FastReducer(uint[] modulus)
             {
                 Debug.Assert(modulus != null);
 
-                // Let r = 4^k, with 2^k > m
+                // Let r = (2^32)^(2k), with (2^32)^k > m
```

Comment:
Could you explain why `base` is `4294967296`?

Reply:
I'd prefer to keep this inline with the paper and just use `2^(k-32)`. We can leave a comment that its 32 because we operate on 32-bits at a time